tests/containers/bci_prepare.pm

```diff
@@ -27,22 +27,29 @@ use testapi;
 
 
 sub packages_to_install {
-    my $host_version = get_required_var('HOST_VERSION');
-    $host_version =~ s/-SP/./;
+    my ($version, $sp, $host_distri) = get_os_release;
```

Comment:
it seems that `packages_to_install` could use the outer `get_os_release`. wdyt?

Reply:
done